public/css/calendar.less

```diff
@@ -183,6 +188,10 @@
   .grid,
   .overlay {
     border-left: none;
+
+    .title {
+      width: 100%;
```

Comment:
This lets the description begin below the title, needlessly. (Since the end time isn't visible anyway)